lib/recognize-stream.ts

```diff
@@ -177,6 +177,8 @@ class RecognizeStream extends Duplex {
       'redaction',
       'end_of_phrase_silence_time',
       'split_transcript_at_phrase_end',
+      'speech_detector_sensitivity',
```

Comment:
we need to add these here: https://github.com/watson-developer-cloud/node-sdk/blob/35fb2a5c9692c6e8a67f9de969269a8d37eb2ebb/lib/recognize-stream.ts#L105

Reply:
Good catch, thanks